### ISSUE TITLE
Question about the use of HTML id 

### ISSUE THREAD

**Issue by Swizz** [KoRiGaN/Vue2Leaflet#24]
Hi,

We are using Vue2Leaflet inside a professional SPA, where maps are literally everywhere and our main concern. 

And, for the map, you choose the following HTML implementation : 
```html
<div id="map">
  <slot></slot>
</div>
```

Is there a real value-added by the following use ?
This id seem to be present only for the use of css, isnt it ?

Can you consider moving to the following one (or similar) ?
```html
<div class="Vue2Leaflet_Map">
  <slot></slot>
</div>
```

Or IDK if the following style implementation work (need some test)
```html
<style scoped>
:first-of-type {
   height: 100%;
   width: 100%;
}
</style>
```

Cheers,

**Comment by KoRiGaN** [comment]
Hi @Swizz,

This id was just used for css style, so I change it to something less common: 'vue2leaflet-map'.

This will be available in the next release.

**Comment by Swizz** [comment]
This issue was more for ask the use of a class name instead of an id, in fact.

**Comment by KoRiGaN** [comment]
as it's only use for css I will use class name instead.

**Comment by KoRiGaN** [comment]
Implemented in v0.0.45
